changelog/@unreleased/pr-5653.v2.yml

```diff
@@ -0,0 +1,5 @@
+type: feature
+feature:
+  description: We now expose an endpoint that kills TimeLock server.
```

Comment:
This is true. I don't know if we want to call it out as loudly in the release notes though!